README.md

```diff
@@ -269,6 +269,21 @@ await $`pwd` // => /home/path
 assert($.verbose == true)
 ```
 
+Building profiled commands stack:
```

Comment:
Now sure, what do you want to show this this example?

Reply:
It's how to create reusable cmd presets.